stock_picking_show_return/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2014-2017 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Show returns on stock pickings",
+    "version": "12.0.1.0.0",
+    "author": "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "category": "Warehouse Management",
+    "website": "https://www.tecnativa.com",
```

Comment:
Should be `https://github.com/OCA/stock-logistics-workflow`

Reply:
Done